html/index.html

```diff
@@ -340,10 +340,10 @@ <h4 class="modal-title">WebSocket connection lost, attempting to reconnect...</h
             </div>
         </footer>
 
-        <!--
         <link rel="stylesheet" href="css/treejs.css" id="treejs_styles">
```

Comment:
These should be handled by gulp and rolled into the main css file. Do they need to be uncommented here?